File: ProblemSet1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void) 
{
    int n;
    do
    {
        n = get_int("Height Positive Integer between 1 and 8: ");
    }
    while (n < 1 || n > 8);
    for (int i = 0; i < n; i++)
    {
        for (int j = (n-1); j > i; j--)
        {
            printf(" ");
        }
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }
        printf("  ");
        for (int h = 0; h <= i; h++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: Problemset6/sentimental-mario-more/mario.py
from cs50 import get_int

# TODO
while True:
    h = get_int("Height - Positive integer between 1 and 8: ")
    if h <= 8 and h >= 1:
        break

for i in range(h):
    print((h - i - 1) * " ", end="")
    print("#" * (i + 1), end="  ")
    print("#" * (i + 1))
